### PR TITLE
Beacon

### PR DESCRIPTION
**Pull request by alisman**

### CODE REVIEW

File: src/appShell/App/Container.tsx [hunk 1]
```diff
@@ -55,6 +54,7 @@ export default class Container extends React.Component<IContainerProps, {}> {
                     </Helmet>
 
                     <div className="pageTopContainer">
+                        <UserMessager dataUrl={"fda"} />
```
Comment (alisman) [inline src/appShell/App/Container.tsx]: this data url is no in use.  right now messages are hardcoded.  will follow up with that

File: src/globalStyles/global.scss [hunk 2]
```diff
@@ -620,3 +620,8 @@ h3.hr {
   color:#ffA500;
   text-decoration: none;
 }
+
```
Comment (kalletlak) [inline src/globalStyles/global.scss]: unnecessary changes

File: src/pages/staticPages/tutorials/Tutorials.tsx [hunk 3]
```diff
@@ -14,6 +14,11 @@ export default class Tutorials extends React.Component<{}, {}> {
             <Helmet>
                 <title>{'cBioPortal for Cancer Genomics::Tutorials'}</title>
             </Helmet>
+            <script>
+                $(document).ready(()=>{
```
Comment (kalletlak) [inline src/pages/staticPages/tutorials/Tutorials.tsx]: this too

File: src/shared/components/userMessage/UserMessage.tsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+        dateEnd:100000000000000,
+        content: `<strong>Announcing the new Group Comparison feature!</strong>: Compare clinical and genomic features of user-defined groups of samples/patients.  <a class='btn btn-primary' href="/tutorials#group-comparison" target="_blank">View Tutorial</a>`,
+        id:'3.0-intro',
+        title:'Announcing the Group Comparison feature of cBioPortal 3.0!'
+    }
+];
+
+
+@observer
+export default class UserMessager extends React.Component<{ dataUrl?:string }, {}> {
```
Comment (kalletlak) [inline src/shared/components/userMessage/UserMessage.tsx]: typo

File: src/shared/components/infoBeacon/styles.module.scss [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    opacity:1;
+    height:$size;
+    width:$size;
+    border-radius:50%;
+    -webkit-transform:translateX(-50%) translateY(-50%);
+    box-shadow: 5px 5px 5px 0px rgba(0,0,0,0.75);
+    animation:beaconActive 2s infinite ease;
+}
+
+//.beacon:before{
```
Comment (kalletlak) [inline src/shared/components/infoBeacon/styles.module.scss]: remove unused

File: src/pages/studyView/StudyViewPage.tsx [hunk 6]
```diff
@@ -155,40 +157,54 @@ export default class StudyViewPage extends React.Component<IStudyViewPageProps,
...
+                                            <SocialAuthButton appStore={this.props.appStore}/>
+                                        </div>
+                                    </If>
+                                </span>)
+                            }
+                        </div>
+                    }
+                >
+                    <button className={classNames('btn btn-primary btn-xs', {active:this.showGroupsTooltip})}
+                            id={"aaron"}
```
Comment (kalletlak) [inline src/pages/studyView/StudyViewPage.tsx]: may be a relevant name
